ui/client/components/AddProcessDialog.tsx

```diff
@@ -29,9 +30,13 @@ export function AddProcessDialog(props: AddProcessDialogProps): JSX.Element {
     async () => {
       if (isValid) {
         const {processId, processCategory} = value
-        await HttpService.createProcess(processId, processCategory, isSubprocess)
-        passProps.close()
-        history.push(visualizationUrl(processId))
+        try {
+          await HttpService.createProcess(processId, processCategory, isSubprocess)
+          passProps.close()
+          history.push(visualizationUrl(processId))
+        } catch(error) {
```

Comment:
all errors should be handled that way? timeout and 5xx as well?

Reply:
I mean - other errors should be IMO rethrowed

Reply:
Yes, we need to rethrow other errors here...